specification/cognitiveservices/data-plane/Personalizer/preview/v2022-09-01-preview/Personalizer.json

```
@@ -0,0 +1,2847 @@
+					"maxLength": 256,
+					"type": "string"
+				},
+				"arguments": {
+					"description": "Arguments of the learning settings.",
+					"maxLength": 1024,
+					"type": "string"
+				}
+			}
+		},
+		"PolicyReferenceContract": {
+			"description": "Reference to the policy within the evaluation.",
+			"required": [
+				"evaluationId",
+				"policyName"
+			],
+			"type": "object",
+			"properties": {
+				"evaluationId": {
+					"description": "Evaluation Id of the evaluation.",
+					"maxLength": 256,
+					"type": "string"
+				},
+				"policyName": {
+					"description": "Name of the learning settings.",
+					"maxLength": 256,
+					"type": "string"
+				}
+			}
+		},
+		"Summary": {
+			"type": "object",
+			"properties": {
+				"expectedReward": {
+					"format": "float",
+					"description": "Average reward.",
+					"type": "number",
+					"readOnly": true
+				},
+				"confidenceInterval": {
+					"description": "Confidence interval.",
+					"type": "array",
+					"items": {
+						"format": "float",
+						"type": "number"
+					},
+					"readOnly": true
+				}
+			}
+		},
+		"Data": {
+			"type": "object",
+			"properties": {
+				"timeStamp": {
+					"format": "date-time",
+					"description": "Timestamp of the aggregation.",
+					"type": "string",
+					"readOnly": true
+				},
+				"expectedReward": {
+					"format": "float",
+					"description": "Average reward.",
+					"type": "number",
+					"readOnly": true
+				},
+				"confidenceInterval": {
+					"description": "Confidence interval.",
+					"type": "array",
+					"items": {
+						"format": "float",
+						"type": "number"
+					},
+					"readOnly": true
+				}
+			}
+		},
+		"TimeSeries": {
+			"type": "object",
+			"properties": {
+				"data": {
+					"description": "Time series.",
+					"type": "array",
+					"items": {
+						"$ref": "#/definitions/Data"
+					},
+					"readOnly": true
+				}
+			}
+		},
+		"PolicyResult": {
+			"type": "object",
+			"properties": {
+				"name": {
+					"description": "The name of the Learning Settings.",
+					"type": "string",
+					"readOnly": true
+				},
+				"slotId": {
+					"description": "Slot id. Empty for overall results and single slot results.",
+					"type": "string",
+					"readOnly": true
+				},
+				"arguments": {
+					"description": "The arguments of the Learning Settings.",
+					"type": "string",
+					"readOnly": true
+				},
+				"policySource": {
+					"description": "The source of the Learning Settings.",
+					"enum": [
+						"Online",
+						"Baseline",
+						"Random",
+						"Custom",
+						"OfflineExperimentation"
+					],
+					"type": "string",
+					"readOnly": true,
+					"x-ms-enum": {
+						"name": "PolicySource",
+						"modelAsString": true
+					}
+				},
+				"summary": {
+					"allOf": [
+						{
+							"$ref": "#/definitions/Summary"
+						}
+					],
+					"readOnly": true
+				},
+				"timeseries": {
+					"allOf": [
+						{
+							"$ref": "#/definitions/TimeSeries"
+						}
+					],
```

Same here -- no `allOf` needed.
```suggestion
					"$ref": "#/definitions/TimeSeries"
```

---

Addressed